tests/test_solido.py

```diff
@@ -208,47 +209,34 @@ def approve_and_execute(
 }
 assert solido_instance['solido']['reward_distribution'] == {
     'treasury_fee': 5,
-    'validation_fee': 3,
     'developer_fee': 2,
-    'st_sol_appreciation': 90,
+    'st_sol_appreciation': 93,
 }
 
-validator_fee_account_owner = create_test_account(
-    'tests/.keys/validator-token-account-key.json'
-)
-
-print(f'> Validator token account owner: {validator_fee_account_owner}')
-
-# Create SPL token
-fee_account = create_spl_token_account(
-    f'tests/.keys/validator-token-account-key.json', st_sol_mint_account
-)
-print(f'> Validator stSol token account: {fee_account}')
-
 
 class Validator(NamedTuple):
     account: TestAccount
     vote_account: TestAccount
-    fee_account: str
+    withdrawer_account: TestAccount
```

Comment:
I think I mentioned this elsewhere already, but we can also make the authorized withdrawer equal to the vote account itself, that should simplify the tests a bit.

Reply:
Yes, you mentioned it in `util.py` but I answer in there